magento/adobe-stock-integration
# 1543: Uncaught JS Error after authorization failure on licensing image from media gallery

### Description (*)

This PR will introduce the fix for the authorization failure on licensing image from the media gallery.
### Fixed Issues (if relevant)

1. Fixes magento/adobe-stock-integration https://github.com/magento/adobe-stock-integration/issues/1543: Uncaught JS Error after authorization failure on licensing image from media gallery
2. ...

### Manual testing scenarios (*)

1. Go to **Content - Media Gallery**, click **Search Adobe Stock**
2. Save an _Unlicensed image_ Preview
 _(the image appeared in Media Gallery)_
3. Go to **Stores - Configuration - Advanced - System - Adobe Stock Integration** and remove the **API Key (Client ID)**. **Save Config**.
4. Go to **Content - Media Gallery** and select the previously saved Unlicensed image
5. Click "three dots" and select **License**
6. Adobe error is displayed in an opened window, after closing the window no additional alerts of console errors are displayed
